Bottom: 1a4e70facebc0023bd254add4af04f4f08501d0b
Top:    0e61c8b68bc90785f846ef5b3729cea9395b4d99
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-28 17:44:43 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
